Makefile

```diff
@@ -9,3 +9,10 @@ clean:
 
 e2e:
 	go test -v --count=1 --tags=e2e ./e2e
+
+push:
+	echo "$(shell cat devenv/state/registry.txt)/app-routing-operator:$(shell date +%s)" > devenv/state/operator-image-tag.txt
+	az acr login -n `cat devenv/state/registry.txt`
+	docker build -t `cat devenv/state/operator-image-tag.txt` .
+	docker push `cat devenv/state/operator-image-tag.txt`
+	kubectl set image --kubeconfig devenv/state/kubeconfig deployments/app-routing-operator operator=`cat devenv/state/operator-image-tag.txt`
```

Comment:
What is the advantage of using files here? Any particular reason why we can't use env vars?

Reply:
This was just to make troubleshooting easier, but I don't mind changing it back to env vars if you prefer

Reply:
files are fine I just wanted to know if there was a specific reason to prefer them over env vars :)